app/src/main/res/layout/activity_benchmark_add_photo.xml

```diff
@@ -84,10 +84,10 @@
         android:layout_marginTop="156dp"
         android:layout_marginEnd="1dp"
         android:background="#00000000"
-        app:icon="@drawable/left"
+        app:icon="@drawable/left_array"
         app:iconTint="@color/black"
         app:layout_constraintEnd_toStartOf="@+id/benchSelectedImage"
-        app:layout_constraintHorizontal_bias="0.4"
+        app:layout_constraintHorizontal_bias="0.529"
```

Comment:
Why are we relying on some random value? Change to dp, or make relative to other components.

Reply:
https://www.google.com/search?q=what+is+bias+in+constraint+layout+android&oq=what+is+constraint+bias&aqs=chrome.1.69i57j0i22i30l2.6492j0j7&sourceid=chrome&ie=UTF-8